backend/DAO/matchDAO.js

```diff
@@ -53,5 +53,25 @@ module.exports = {
                     throw new SuperError(ERR_INTERNAL_SERVER_ERROR, 'There has been a problem retrieving your matches. Please try again')
                 }
             })
+    },
+
+    deleteMatch: (studentId, placementId) => {
+        return database('student_has_placement as shp')
+            .where('student_id', studentId)
+            .andWhere('placement_id', placementId)
+            .del();
+    },
```

Comment:
In theory this can also throw a server error, no?

Reply:
you mean in case of no matches? It returns an empty list in that case.